packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -923,6 +986,13 @@ export class Repository extends Construct implements IRepository {
       version.linuxFullVersionString(),
     ];
 
+    if (this.secretsManagementSettings.enabled) {
+      installerArgs.push('true');
```

Comment:
This will need to be updated after a rebase. The installer script has been changed to use named parameters instead of positional parameters